projects/crypto-currency/destemidos/App/src/components/Feeds/PostCardFull.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import CardHeader from '@material-ui/core/CardHeader'
+import CardContent from '@material-ui/core/CardContent'
+import CardActions from '@material-ui/core/CardActions'
+import IconButton from '@material-ui/core/IconButton'
+import Typography from '@material-ui/core/Typography'
+import FavoriteIcon from '@material-ui/icons/Favorite'
+import ShareIcon from '@material-ui/icons/Share'
+import TwitterIcon from '@material-ui/icons/Twitter'
+import {CardDiv, CardMediaDiv, StyledAvatar, TypographyDiv} from './Style.js'
```

Comment:
Poderia colocar espaço no começo e no final das chaves?
```suggestion
import { CardDiv, CardMediaDiv, StyledAvatar, TypographyDiv } from './Style.js'
```

Reply:
alterado